src/main/app/src/app/zaken/zaken.service.ts

```diff
@@ -224,4 +234,82 @@ export class ZakenService {
         );
     }
 
+    addTeKoppelenZaak(zaak: Zaak): void {
```

Comment:
De koppelfunctionaliteit die geen REST calls doet naar de backend, zoals deze methode,  graag in een aparte service stoppen. ZakenKoppelService of zo.